### PR TITLE
Update to node 8.3-pre WIP

### PR DESCRIPTION
**Pull request by kderme**
# Description

Add your description here, if it fixes a particular issue please provide a [link](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword=) to the issue.

# Checklist

- [ ] Commit sequence broadly makes sense
- [ ] Commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [ ] Any changes are noted in the [changelog](https://github.com/input-output-hk/cardano-db-sync/blob/master/cardano-db-sync/CHANGELOG.md)
- [ ] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) on version 0.10.1.0 (which can be run with `scripts/fourmolize.sh`)
- [ ] Self-reviewed the diff

# Migrations

- [ ] The pr causes a [breaking change](https://github.com/input-output-hk/cardano-db-sync/blob/master/doc/migrations.md) of type a,b or c
- [ ] If there is a breaking change, the pr includes a database migration and/or a fix process for old values, so that upgrade is possible
- [ ] Resyncing and running the migrations provided will result in the same database semantically

If there is a breaking change, especially a big one, please add a justification here. Please elaborate
more what the migration achieves, what it cannot achieve or why a migration is not possible.

### CODE REVIEW

File: doc/installing.md [hunk 1]
```diff
@@ -127,6 +129,60 @@ make check
 sudo make install
 ```
 
+### Installing blst
```
Comment (Cmdv) [inline doc/installing.md:132]: General question, is this a hard requirement to run db-sync moving forwards?

**Review by Cmdv — DISMISSED** [review]
LGTM 👍 
Though there still seems to be a bunch of build errors 😞

File: doc/installing.md (hunk 1 above)
Comment (kderme) [reply doc/installing.md:132]: Yes it doesn't build otherwise. This dependency comes from `cardano-crypto-class`

**Comment by kderme** [comment]
> LGTM 👍 Though there still seems to be a bunch of build errors 😞

Yes some build errors in tests remain

File: doc/installing.md (hunk 1 above)
Comment (Cmdv) [reply doc/installing.md:132]: Thanking you both 👍

**Comment by angerman** [comment]
- Buildkite was removed in #1497, so we can probably ignore that.
- The GHA failures look like some incorrect plutus is chosen.
- So do the hydra jobs.
- The Haddock jobs seem to fail on `BangPatterns` 🤦

**Comment by angerman** [comment]
FYI: We could also simplify the installation instructions using input-output-hk/devx (for those who have nix installed), it will ensure that you get all the necessary system libraries, and then you can use `cabal`.

**Comment by kderme** [comment]
> * Buildkite was removed in [Remove Buildkite jobs #1497](https://github.com/input-output-hk/cardano-db-sync/pull/1497), so we can probably ignore that.
> * The GHA failures look like some incorrect plutus is chosen.
> * So do the hydra jobs.
> * The Haddock jobs seem to fail on `BangPatterns` 🤦

I haven't updated the tests yet, that's why they fail. I don't think an incorrect plutus version is chosen.

**Comment by kderme** [comment]
Dropping the srp and updating chap caused an extended change to used dependencies. So I opted to simply update the srp to node-8.3.1-pre. We will try to find a better solution once we want to release for mainnet.

```
$ diff cabal.project.freeze cabal.project.freeze-1-1-0
33c33
<              any.ansi-wl-pprint ==0.6.9,
---
>              any.ansi-wl-pprint ==1.0.2,
93c93
<              any.cardano-api ==8.12.0.0,
---
>              any.cardano-api ==8.16.3.0,
107,112c107,112
<              any.cardano-git-rev ==0.1.2.0,
<              any.cardano-ledger-allegra ==1.2.1.0,
<              any.cardano-ledger-alonzo ==1.3.3.0,
<              any.cardano-ledger-alonzo-test ==1.1.2.2,
<              any.cardano-ledger-api ==1.3.1.0,
<              any.cardano-ledger-babbage ==1.4.2.0,
---
>              any.cardano-git-rev ==0.1.3.0,
>              any.cardano-ledger-allegra ==1.2.1.1,
>              any.cardano-ledger-alonzo ==1.4.0.0,
>              any.cardano-ledger-alonzo-test ==1.1.2.3,
>              any.cardano-ledger-api ==1.4.0.0,
>              any.cardano-ledger-babbage ==1.4.3.0,
114c114
<              any.cardano-ledger-binary ==1.1.1.1,
---
>              any.cardano-ledger-binary ==1.1.2.0,
118,120c118,120
<              any.cardano-ledger-conway ==1.6.3.0,
<              any.cardano-ledger-conway-test ==1.2.0.3,
<              any.cardano-ledger-core ==1.4.1.0,
---
>              any.cardano-ledger-conway ==1.7.1.0,
>              any.cardano-ledger-conway-test ==1.2.0.4,
>              any.cardano-ledger-core ==1.5.0.0,
122c122
<              any.cardano-ledger-mary ==1.3.1.0,
---
>              any.cardano-ledger-mary ==1.3.2.0,
124,125c124,125
<              any.cardano-ledger-pretty ==1.2.1.2,
<              any.cardano-ledger-shelley ==1.4.2.0,
---
>              any.cardano-ledger-pretty ==1.3.0.0,
>              any.cardano-ledger-shelley ==1.5.1.0,
127,129c127,129
<              any.cardano-ledger-shelley-ma-test ==1.2.1.1,
<              any.cardano-ledger-shelley-test ==1.2.0.2,
<              any.cardano-node ==8.2.1,
---
>              any.cardano-ledger-shelley-ma-test ==1.2.1.2,
>              any.cardano-ledger-shelley-test ==1.2.0.3,
>              any.cardano-node ==8.3.0,
135c135
<              any.cardano-protocol-tpraos ==1.0.3.3,
---
>              any.cardano-protocol-tpraos ==1.0.3.4,
246,247c246,247
<              any.fs-api ==0.1.0.3,
<              any.fs-sim ==0.2.0.0,
---
>              any.fs-api ==0.2.0.0,
>              any.fs-sim ==0.2.1.0,
306d305
<              any.int-cast ==0.2.0.0.0.0.0.0.1,
397,401c396,400
<              any.ouroboros-consensus ==0.9.0.0,
<              ouroboros-consensus -asserts -expose-sublibs,
<              any.ouroboros-consensus-cardano ==0.7.0.0,
<              ouroboros-consensus-cardano -asserts -expose-sublibs,
<              any.ouroboros-consensus-diffusion ==0.7.0.0,
---
>              any.ouroboros-consensus ==0.10.0.1,
>              ouroboros-consensus -asserts,
>              any.ouroboros-consensus-cardano ==0.8.0.1,
>              ouroboros-consensus-cardano -asserts,
>              any.ouroboros-consensus-diffusion ==0.7.1.1,
403c402
<              any.ouroboros-consensus-protocol ==0.5.0.4,
---
>              any.ouroboros-consensus-protocol ==0.5.0.6,
432c431
<              any.plutus-core ==1.7.0.0,
---
>              any.plutus-core ==1.9.0.0,
434,435c433,434
<              any.plutus-ledger-api ==1.7.0.1,
<              any.plutus-tx ==1.7.0.0,
---
>              any.plutus-ledger-api ==1.9.0.0,
>              any.plutus-tx ==1.9.0.0,
445c444,445
<              any.prettyprinter-configurable ==1.7.0.0,
---
>              any.prettyprinter-compat-ansi-wl-pprint ==1.0.2,
>              any.prettyprinter-configurable ==1.9.0.0,
491c491
<              any.set-algebra ==1.1.0.1,
---
>              any.set-algebra ==1.1.0.0,
576,577c576,577
<              any.trace-dispatcher ==2.0.1,
<              any.trace-forward ==2.0.1,
---
>              any.trace-dispatcher ==2.0.3,
>              any.trace-forward ==2.0.2,
642c642
< index-state: cardano-haskell-packages 2023-09-05T08:50:43Z, hackage.haskell.org 2023-08-09T22:19:16Z
---
> index-state: cardano-haskell-packages 2023-09-01T20:45:06Z, hackage.haskell.org 2023-08-09T22:19:16Z
```

**Pull request merged**